### PR TITLE
Add copied files from Microverse trial project to review them.

### PR DESCRIPTION
**Pull request by jchernandez87**
I copied the files from my microverse trials project to make a review of them when making a pull request.

### CODE REVIEW

**Comment by jchernandez87** [comment]
Your project should have a .gitignore file and you should put the node_modules folder in it.

**Comment by jchernandez87** [comment]
You are missing your linters file.

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is my project for the review-your-project-from-trials exercise at microverse.
```
Comment (jchernandez87) [inline README.md:1]: - Your project should have a .gitignore file and you need to put your node_modules folder in it.
- Your linters file is missing, you need to set up your linters.

**Review by jchernandez87 — COMMENTED** [review]
# Changes required ♻️ 

-You are getting linters errors, Try to fix them.

😃

**Review by christianotieno — APPROVED** [review]
Hi @jchernandez87, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**